src/modules/cron/cron.controller.ts

```diff
@@ -178,4 +183,49 @@ export class CronController {
             throw new Error('Failed to remove users due to an internal server error.');
         }
     }
+
+    @Put('unlock')
+    @ApiOkResponse({
+        description: 'The users were successfully unlocked.',
+        type: Boolean,
+    })
+    @ApiUnauthorizedResponse({ description: 'Not authorized to unlock users.' })
+    @ApiForbiddenResponse({ description: 'Insufficient permissions to unlock users.' })
+    @ApiNotFoundResponse({ description: 'Insufficient permissions to unlock users.' })
+    @ApiInternalServerErrorResponse({ description: 'Internal server error while trying to unlock users.' })
+    public async unlockUsersWithExpiredLocks(@Permissions() permissions: PersonPermissions): Promise<boolean> {
+        try {
+            const userLocks: UserLock[] = await this.userLockRepository.getLocksToUnlock();
+            if (userLocks.length === 0) {
+                return true;
+            }
+
+            const results: PromiseSettledResult<Result<void, DomainError>>[] = await Promise.allSettled(
+                userLocks.map(async (userLock: UserLock) => {
+                    const person: Result<Person<true>> = await this.personRepository.getPersonIfAllowed(
+                        userLock.person,
+                        permissions,
+                    );
+                    if (!person.ok || !person.value.keycloakUserId) {
+                        return { ok: false, error: new EntityNotFoundError() };
```

Comment:
I was gonna request a change here to return a more descriptive error and provide feedback on which user can't be unlocked but seems like all the endpoints just throw a generic error and say it's an internal server error... Not so good but I'll just approve it since it's done this way everywhere and this is only used for cron jobs.

Reply:
The reasoning here was, that this code is only used for cron jobs. It might be a good idea, that we extend it in the future, but right now it's good to just get the functionality with prio.